scaletest/dashboard/chromedp.go

```diff
@@ -209,6 +230,26 @@ func visitMainPage(ctx context.Context, u *url.URL) error {
 	return chromedp.Run(ctx, chromedp.Navigate(u.String()))
 }
 
+func Screenshot(ctx context.Context, name string) (string, error) {
+	var buf []byte
+	if err := chromedp.Run(ctx, chromedp.CaptureScreenshot(&buf)); err != nil {
+		return "", xerrors.Errorf("capture screenshot: %w", err)
+	}
+	fname := fmt.Sprintf("scaletest-dashboard-%s-%s.png", name, time.Now().Format("20060102-150405"))
```

Comment:
is there a chance in the future that 2 screenshots can be taken at the same time?

Reply:
Ok, I see that the probability is relatively low 👍 Thanks for clarifying!